clients/context-registry/src/ContextRegistryClient.ts

```diff
@@ -6,8 +6,9 @@
  * @module ContextRegistry
  */
 import * as deepAssign from "deep-assign";
-import { assert } from "@bentley/bentleyjs-core";
-import { AuthorizedClientRequestContext, ECJsonTypeMap, RequestOptions, RequestQueryOptions, WsgClient, WsgInstance } from "@bentley/itwin-client";
+import { AuthorizedClientRequestContext, RequestOptions, RequestQueryOptions, } from "@bentley/itwin-client";
+import { ECJsonTypeMap, WsgInstance } from "./wsg/ECJsonTypeMap";
+import { WsgClient } from "./wsg/WsgClient";
```

Comment:
do you need to export any of these wsg modules in the barrel for this pkg?

Reply:
They're not used directly for the context registry and I export them from iModelhub client due to them being used in that context. I didn't really worry too much about it since they'll only be used in test code going forward and the important part was getting it all out of the public APIs. Then I'll circle back and clear it all up.